client/my-sites/people/team-list/team.jsx

```diff
@@ -0,0 +1,151 @@
+/**
+ * External dependencies
+ */
+var React = require( 'react' ),
+	debug = require( 'debug' )( 'calypso:my-sites:people:team-list' ),
+	omit = require( 'lodash/omit' );
+
+/**
+ * Internal dependencies
+ */
+var Card = require( 'components/card' ),
+	PeopleListItem = require( 'my-sites/people/people-list-item' ),
+	UsersActions = require( 'lib/users/actions' ),
+	InfiniteList = require( 'components/infinite-list' ),
+	deterministicStringify = require( 'lib/deterministic-stringify' ),
+	NoResults = require( 'my-sites/no-results' ),
+	analytics = require( 'lib/analytics' ),
+	PeopleListSectionHeader = require( 'my-sites/people/people-list-section-header' ),
+	config = require( 'config' );
+
+module.exports = React.createClass( {
+	displayName: 'Team',
```

Comment:
As long as we assign our React classes to a binding name then we won't need to manually create `displayName`

```js
const Team = React.createClass( …

const Team = props => …

export class Team extends Component { …
```